posthog-react-native/src/hooks/useLifecycleTracker.ts

```diff
@@ -2,18 +2,24 @@ import { useEffect, useRef } from 'react'
 import { AppState } from 'react-native'
 import type { PostHog } from '../posthog-rn'
 import { usePostHog } from './usePostHog'
+import { Linking } from 'react-native'
+
+async function captureApplicationOpened(posthog: PostHog): Promise<void> {
+  const url = (await Linking.getInitialURL()) ?? undefined
+  posthog.capture('Application Opened', { url })
+}
 
 export function useLifecycleTracker(client?: PostHog): void {
-  const openTrackedRef = useRef(false)
+  const openTrackedRef = useRef<boolean>(false)
```

Comment:
https://github.com/PostHog/posthog-js-lite/blob/ac83f9a98806e7a4c2c32d38ea874cc31c41c645/posthog-react-native/src/frameworks/wix-navigation.ts#L20-L40
does something very similar, we could delete that function, extract and reuse the `useLifecycleTracker` in the wix navigation?

Reply:
another issue is that `useLifecycleTracker` isn't being used anywhere?
can we actually get rid of this now (and trackLifecycle) since its all controlled via `captureNativeAppLifecycleEvents`

Reply:
Yeah, happy to update `wix-navigation` to use that hook. The reason why I didnt remove the hook is that it's exposed, so someome might be using it directly without our knowledge, so I thought of keeping it around until the next major.

Reply:
i think its fine because we dont document `useLifecycleTracker` anywhere so its safe to remove since we also have an alternative, same for `trackLifecycle`